### ISSUE TITLE
Hit ENTER in New File dialog throws error

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2573]
Error when creating new file by hitting ENTER when dialog appears:

```
Uncaught TypeError: Cannot read property 'value' of undefined
```

Seen on Maqetta+Orion build.

**Comment by peller** [comment]
related to #2528?

**Comment by doronrosenberg** [comment]
fixed in rev da87f49abf81a576c10591218fe3e3b39121346d
